src/main/kotlin/org/rust/openapiext/psi.kt

```diff
@@ -137,12 +143,19 @@ private class RsWithMacrosRecursiveElementWalkingVisitor(
         }
     }
 
+    private fun tryProcessAttrProcMacro(element: PsiElement): Boolean {
+        if (element !is RsAttrProcMacroOwner) return false
+        val attr = element.procMacroAttributeWithDerives as? ProcMacroAttribute.Attr ?: return false
```

Comment:
Can we call `procMacroAttributeWithDerives` only one time for an item, as we do in `RsElement.processItem`?
Currently, `procMacroAttributeWithDerives` is called second time in `tryProcessDeriveProcMacro` .